README.md

```diff
@@ -23,6 +23,41 @@ scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
 scoop install platform
 ```
 
+### Bash installer
+
+Use the bash installer for an automated installation, using the most preferable way for your system.
+
+```console
+curl -fSsl https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
```

Comment:
For vanity and for the GitHub rate limit reason mentioned below, we should host this on platform.sh.

https://platform.sh/cli/installer is taken so we'd have to come up with a different path (perhaps just `/cli/installer.sh`)

Reply:
Good catch, added support for the environment variable in the latest commit

https://github.com/platformsh/cli/blob/68ece4d3b440e9e26a6da00b1de5d3b8b72efd2a/installer.sh#L149-L152

Reply:
It still has a stray `-l` and we still need to host it I think